salt/master.py

```diff
@@ -103,6 +103,12 @@
 log = logging.getLogger(__name__)
 
 
+def _make_master_daemon_runner_client(opts):
+    ropts = dict(opts)
+    ropts["quiet"] = True
```

Comment:
I feel like this should be `ropts.setdefault("quiet", True)`, otherwise it's going to override the option. Unless that's the intention 🤔

Reply:
Certainly that was the behaviour of the code that I was inspired by (line 198), but I don't know if that was the intention of that code. I thought it best to be consistent in the new place that I was applying this to. But if we're happy to change the behaviour for that bit. Then what you suggest works for me. Or maybe there's a reason that's not visible to me that means that we actually want different behaviour in these two places.

Reply:
Well, we're on the same page there, then 😂 I'm inclined towards consistency, I'd just like to know the original /why/, to know whether or not it's a foolish consistency 😝

Reply:
I'm going to take a punt at some logic to support the way that this was and therefore keeping this the same and making the other place where we create a RunnerClient consistent with the first:

The quiet flag seems only to be used to control the main human facing output for the RunnerClient this is the result of the salt runner that you are executing. It obviously makes sense to display this when the RunnerClient is being used from inside salt-run. However when the RunnerClient is being used from with in the salt-master it never make sense to display this because the salt-master is acting as a server and what ever client initiated the request is responsible for displaying this to the user. Therefore the RunnerClient should always be used with quiet=False in the salt-master.